### PR TITLE
#160577469 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by gloriaodipo**
**What does this PR do?**
Integrate TravisCI with readme badge for continuous integration.

**Description of Task to be completed**
- Create a .travis.yml file and add setup for TravisCI.
- Link repo with TravisCI and add badge showing the build status in README.md.

**What are the relevant PT stories?**
[160577469](https://www.pivotaltracker.com/n/projects/2198533/stories/160577469)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+language: python
+
+python:
+    - "3.6"
```
Comment (FawazFarid) [inline .travis.yml]: indentation is a bit off here

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+services:
+    - postgresql
```
Comment (FawazFarid) [inline .travis.yml]: ditto

**Review by ibetween — APPROVED** [review]
I have reviewed the changes I approve them

File: README.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+[![Build Status](https://travis-ci.org/andela/ah-shakas.svg?branch=develop)](https://travis-ci.org/andela/ah-shakas)
```
Comment (FawazFarid) [inline README.md]: Can we change this to `master`, I'm gonna explain later on Slack channel why

File: authors/.travis.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+language: python
```
Comment (FawazFarid) [inline authors/.travis.yml]: why do we have this file here?

**Review by FawazFarid — CHANGES_REQUESTED** [review]

File: authors/.travis.yml (hunk 4 above)
Comment (gloriaodipo) [reply authors/.travis.yml]: I made a mistake when creating the travis file, it should'nt be in this position. I'll remove it.

File: README.md (hunk 3 above)
Comment (gloriaodipo) [reply README.md]: Sure.

**Review by FawazFarid — APPROVED** [review]
Looks good to me, thanks

**Pull request merged**